### PR TITLE
Patch build error

### PR DESCRIPTION
**Pull request by tuncK**
When using the recent upstream image, the container's conda otherwise says "inconsistent" and takes a very long time to respond.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,31 +1,50 @@
...
-# Python packages
+# Set channels to (defaults) > bioconda > conda-forge
 RUN conda config --add channels conda-forge && \
-    conda config --add channels bioconda && \
+    conda config --add channels bioconda
+    #conda config --add channels defaults
+
+# Pre-installed mamba is raising a conda error:
+# "The environment is inconsistent"
+RUN conda remove mamba --yes
```
Comment (bgruening) [inline Dockerfile:17]: Oh, this is unfortunate ...

Comment (bgruening) [reply Dockerfile:17]: Is it worthwhile to try python-3.11 https://hub.docker.com/r/jupyter/datascience-notebook/tags?page=1&name=python-3.1

There is the potential that it creates other python issues or a few conda packages are not available.
What about updating mamba instead?

**Review by bgruening — APPROVED** [review]
Either way, feel free to merge and proceed.

File: Dockerfile (hunk 1 above)
Comment (tuncK) [reply Dockerfile:17]: I am essentially updating mamba, but conda update did not work smoothly. So I am deleting mamba here, and re-installing the newer version at line #39.

I already tried Python 3.11, it works out of the box. But some libs/programs might not yet be compatible with it, so I want to make a 3.10 release before 3.11.

**Comment by bgruening** [comment]
Thanks @tuncK

**Pull request merged**